manifests/plugin/dynflow.pp

```diff
@@ -14,22 +14,54 @@
 # $console_auth::    Whether to enable trusted hosts and ssl for the dynflow console
 #                    type:boolean
 #
+# $core_port::       Port to use for the local dynflow core service
+#
 class foreman_proxy::plugin::dynflow (
   $enabled           = $::foreman_proxy::plugin::dynflow::params::enabled,
   $listen_on         = $::foreman_proxy::plugin::dynflow::params::listen_on,
   $database_path     = $::foreman_proxy::plugin::dynflow::params::database_path,
   $console_auth      = $::foreman_proxy::plugin::dynflow::params::console_auth,
+  $core_port         = $::foreman_proxy::plugin::dynflow::params::core_port,
 ) inherits foreman_proxy::plugin::dynflow::params {
 
   validate_bool($enabled, $console_auth)
   validate_listen_on($listen_on)
   validate_absolute_path($database_path)
 
+  $use_ssl = $foreman_proxy::ssl
+
+  if $use_ssl {
+    $core_url = "https://${::fqdn}:${core_port}"
+  } else {
+    $core_url = "http://${::fqdn}:${core_port}"
+  }
+
   foreman_proxy::plugin { 'dynflow':
   } ->
   foreman_proxy::settings_file { 'dynflow':
     enabled       => $enabled,
     listen_on     => $listen_on,
     template_path => 'foreman_proxy/plugin/dynflow.yml.erb',
   }
+
+  if $::osfamily == 'RedHat' and $::operatingsystem != 'Fedora' {
+    $scl_prefix = 'tfm-'
+  } else {
+    $scl_prefix = '' # lint:ignore:empty_string_assignment
+  }
+
+  # Currently the service is only needed on Red Hat OS's
```

Comment:
It will be once I finish the packaging Debian, but the service will still only be needed on Red Hat because it's separate from the system ruby in SCL.